### DISCUSSION TITLE
Set values for elements in GRUB_BTRFS_TITLE_FORMAT

### DISCUSSION THREAD

**Discussion by sid115** [Antynea/grub-btrfs#291] [Q&A]
How do I set values for elements in GRUB_BTRFS_TITLE_FORMAT in `/etc/default/grub-btrfs/config`, especially "type" and "description", when making a snapshot so that they are displayed in the Grub menu?

Currently, the values for "type" and "description" are "N/A" when I take snapshots manually or using [btrbk](https://github.com/digint/btrbk).

**Comment by Schievel1** [comment]
These values are set during snapshot creation. 
They are not the business of grub-btrfs but rather of the programs that creates the snapshots.
